src/Service/Sso/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Added
 
 - Added region `il-central-1`
+- AWS api-change: TODO
```

Comment:
Needs updating.

Reply:
good catch, addressed in addressed in https://github.com/async-aws/aws/pull/1572